### DISCUSSION TITLE
Add Local Sizing in Watertight Geometry workflow

### DISCUSSION THREAD

**Discussion by mcgoldba** [ansys/pyfluent#2219] [Q&A] [answered]
How do I add label specific local sizing with pyFluent?  

I am attempting to modify the mixing elbow example in the documentation   (https://fluent.docs.pyansys.com/version/stable/examples/00-fluent/mixing_elbow.html#sphx-glr-examples-00-fluent-mixing-elbow-py), so that local sizing is added for e.g. the 'wall-inlet' boundary.  I have tried with

```
add_local_sizing = meshing.workflow.TaskObject["Add Local Sizing"]
add_local_sizing.Arguments = {
    'AddChild': "yes",
    'BOIControlName': 'wall-inlet', 
    'BOIExecution': "Face Size",
    'BOIFaceLabelList': ['wall-inlet'],
    'BOISize': 0.01,
}
add_local_sizing.Execute()
```
which successfully updates the parameters in the workflow dialog, but doesn't actually create the 'wall-inlet' sizing sub-object in the workflow. 

![image](https://github.com/ansys/pyfluent/assets/50269816/81580d7c-7878-4a5d-8f70-d0c1095f1fba)

In the GUI environment, this would be accomplished by clicking the "Add Local Sizing" button, but how do I do this with pyFluent?  As is, no local sizing is used when creating surface mesh.

**Comment by raph-luc** [comment]
I believe you need to use the `add_local_sizing.AddChildToTask()` command before `Execute()`

**Reply by raph-luc** [reply, marked as answer]
Alternatively, I believe `AddChildAndUpdate()` can be used instead of `AddChildToTask()` + `Execute()`

**Reply by mcgoldba** [reply]
`AddChildToTask()` didn't work for me, but `AddChildAndUpdate()` did.  Thanks!
